hrms/hr/doctype/expense_claim_detail/expense_claim_detail.json

```diff
@@ -58,8 +58,6 @@
    "read_only": 1
   },
   {
-   "fetch_from": "expense_type.description",
-   "fetch_if_empty": 1,
```

Comment:
fetch if empty makes sure it doesn't overwrite user's changes. This is not the problem

Reply:
https://github.com/frappe/hrms/blob/64039786b9fd8a631592e5dd50dc8b060d26fef2/hrms/hr/doctype/expense_claim/expense_claim.js#L65-L67

remove these lines instead